.golangci.yml

```diff
@@ -33,4 +33,4 @@ linters:
   - canonicalheader   # The headers in http.Request are canonical by default
   - promlinter
   - containedctx
```

Comment:
I wonder why these two linters are disabled :thinking: 

I think we can reactivate them :slightly_smiling_face: (maybe in another PR)

Reply:
I have no idea 🤷 
Let's see later!